scripts/flatten.sh

```diff
@@ -6,14 +6,18 @@ fi
 
 mkdir -p flats/kitty
 mkdir -p flats/simpleKitty
+mkdir -p flats/mediator
+mkdir -p flats/upgradeability
 
 FLATTENER=./node_modules/.bin/truffle-flattener
 KITTY_CONTRACTS_DIR=contracts/kitty
 
 
-echo "Flattening kitty contracts"
 ${FLATTENER} ${KITTY_CONTRACTS_DIR}/KittyCore.sol > flats/kitty/KittyCore_flat.sol
 ${FLATTENER} ${KITTY_CONTRACTS_DIR}/GeneScience.sol > flats/kitty/GeneScience_flat.sol
```

Comment:
Do we need flattened contracts for `GeneScience.sol`, `SaleClockAuction.sol` and `SiringClockAuction.sol`?

Reply:
Actually no. I removed them https://github.com/poanetwork/cryptokitties-xdai-demo/pull/6/commits/594084a7d2a30a1d3f9b43427c1bf320977bb6df